pom.xml

```diff
@@ -338,6 +338,11 @@
                 <artifactId>jackson-annotations</artifactId>
                 <version>${com.fasterxml.jackson.version}</version>
             </dependency>
+            <dependency>
+                <groupId>com.fasterxml.jackson.dataformat</groupId>
+                <artifactId>jackson-dataformat-yaml</artifactId>
+                <version>${com.fasterxml.jackson.version}</version>
+            </dependency>
```

Comment:
I believe we can use snake-yaml instead of this. Shall we use that?

Reply:
Moved into snake yaml.